### ISSUE TITLE
Minified JS doesn't match compressed JS

### ISSUE THREAD

**Issue by JamesSkemp** [kenwheeler/slick#2828]
> The committed version of minified JS does not match the JS file if it were compressed. In particular it's missing accessibility-related fixes.

====================================================================

#### Steps to reproduce the problem
1. View the current version of slick.js at https://github.com/kenwheeler/slick/blob/master/slick/slick.js
2. Search for 'aria-describedby' and note context.
3. View the current version of slick.min.js at https://github.com/kenwheeler/slick/blob/master/slick/slick.min.js
4. Search for 'aria-describedby' and note context.

====================================================================

#### What is the expected behaviour?
The committed version of slick.min.js should include improvements committed to slick.js.

====================================================================

#### What is observed behaviour?
slick.min.js does not include recent merges.

====================================================================

#### More Details
Note also that http://kenwheeler.github.io/slick/ links to https://github.com/kenwheeler/slick/archive/1.6.0.zip which appears to be almost one year old.

**Comment by michaelthorne** [comment]
Agreed. Having a look at when files were last updated pretty much confirms this:

https://github.com/kenwheeler/slick/tree/master/slick

`slick.js | update on ready event listeners to jquery 3 compliant syntax | 3 months ago`
`slick.min.js | 1.6.0 | 11 months ago`

**Comment by leggomuhgreggo** [comment]
So, the minified version is the release version. The unminified is the master, where we merge new PRs for future releases. Should be a new release soon.
